snap-packages/launchers/patch-netbeans-conf

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# If the user has a custom netbeans.conf. Disable Java Security Manager as of NetBeans 25
+if [ -f "$SNAP_USER_DATA/etc/netbeans.conf" ]; then
+    if ! grep -q '^netbeans_default_options=".*-J-Djava.lang.Runtime.level=' $SNAP_USER_DATA/etc/netbeans.conf ; then
+        sed -i 's/netbeans_default_options="/netbeans_default_options="-J-Djava.lang.Runtime.level=FINE /' $SNAP_USER_DATA/etc/netbeans.conf
+    fi
+    if ! grep -q '^netbeans_default_options=".*-J-DTopSecurityManager.disable=' $SNAP_USER_DATA/etc/netbeans.conf ; then
+        sed -i 's/netbeans_default_options="/netbeans_default_options="-J-DTopSecurityManager.disable=true /' $SNAP_USER_DATA/etc/netbeans.conf
+    fi
+fi
```

Comment:
will this run every time someone downloads a new release?

NB 26 will likely remove `-J-DTopSecurityManager.disable=true` again as clean up. The flag would become a no-op (since the code would be gone) - so it is probably ok if it keeps getting added i suppose.

Reply:
This will be executed before every NetBeans start.

When the Security Manager gets removed from NB26, I'd update this one, to have these removed or just leave the custom `netbeans.conf` alone.